src/commands/Fun/destruct.js

```diff
@@ -0,0 +1,61 @@
+exports.run = function(bot,msg,args) {
+
+    let parsedArgs = bot.utils.parseArgs(args, ['d:','s:']);
+
+    if(parsedArgs.leftover.length < 1) {
+        throw 'Please provide a secret message';
+    }
+
+    let message = parsedArgs.leftover.join(' ');
+    let delay = isNaN(parsedArgs.options.d) ? 5000 : parseInt(parsedArgs.options.d);
+    delay = (delay < 100) ? 100 : delay;
+    let style = (typeof parsedArgs.options.s === 'string') ? parsedArgs.options.s : 'plain';
+
+    msg.delete();
+
+    switch(style) {
+    case 'embed':
+        message = {
+            embed: bot.utils.embed(
+                `This message self-destructs in ${delay/1000} seconds.`,
+                message,
+                [],
+                {
+                    inline: true,
+                    footer: 'Secret Message',
+                    color: [255, 0, 0]
```

Comment:
I would prefer to not set the color and leave the embed color handling to bot.utils.embed itself. The reason for this is that I'm planning on writing a settings system, and I want to allow the bot settings to control the default embed color handling. For example, if you wanted all your embeds for the bot to be pink, you could do something like `//settings embedColor #ff00ff`.